brainsets_pipelines/cmi_mipdb_2016/eyetracking.py

```
@@ -0,0 +1,695 @@
+        paradigm_intervals, "event_onsets"
+    ):
+        for paradigm_idx in range(len(paradigm_intervals)):
+            event_codes = parse_comma_separated_numbers(
+                paradigm_intervals.event_codes[paradigm_idx]
+            )
+            event_onsets = parse_comma_separated_numbers(
+                paradigm_intervals.event_onsets[paradigm_idx]
+            )
+            grouped[paradigm_idx] = [
+                (float(onset), int(code))
+                for onset, code in zip(event_onsets, event_codes)
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)